hudi-flink/src/main/java/org/apache/hudi/sink/utils/HiveSyncContext.java

```diff
@@ -51,11 +51,10 @@ public HiveSyncTool hiveSyncTool() {
     return new HiveSyncTool(this.syncConfig, this.hiveConf, this.fs);
   }
 
-  public static HiveSyncContext create(Configuration conf) {
+  public static HiveSyncContext create(SerializableConfiguration serConf, Configuration conf) {
```

Comment:
IMO, we'd better make these two parameters more readable. The first one is serialized `Configuration` for Hadoop. The second one is the requirement of Flink right? While, from the name, we can not distinguish them.